### ISSUE TITLE
Try to use it in a Stencil JS application.

### ISSUE THREAD

**Issue by matteocusinato** [TeamHive/pdf-viewer#30]
Hi,
I'm trying to use it in a Stencil JS application.

I followed the installation guide.

But I have these errors:
![Schermata 2020-07-09 alle 16 54 19](https://user-images.githubusercontent.com/19340970/87055841-233c4980-c205-11ea-8869-137ef2a75916.png)

Can you help me? Thanks

**Comment by JEricaM** [comment]
Same problem here. I'm stuck :S @matteocusinato  have you solved?

**Comment by matteocusinato** [comment]
@JEricaM no, I had resolved in changing without the plugin uploading it in a .jpeg format..

**Comment by goulu** [comment]
Same here :-( 
something is missing in the instructions ...
(I don't want to go back to a server-side rendering like the people above...)

**Comment by Songkeys** [comment]
I have exactly the same issue. @goulu 's solution is not working for me :(
